dosing_pump_controller/dosing_pump_controller.ino

```diff
@@ -38,33 +39,40 @@ Menu pump3_menu("Pump 3");
 Menu settings_menu("Settings");
 
 // Returns control to the main_menu, to be used as the completion for menu items added to the main_menu.
-void mainMenuReturnControl() {
-  main_menu.present();
-}
+void mainMenuReturnControl() { main_menu.present(); }
```

Comment:
This seems fine. We should probably create a style guide and include one liner exceptions like this.